File: week1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height>8||height<1);

    for(int i=0;i<height;i++)
    {
       for(int d= height-1; d>i;d = d-1)
            {
                printf(" ");
            }
       for(int h=-1;h<i;h++)
        {

            printf("#");
        }
        printf("\n");   
    }
}

File: week6/sentimental-mario-less/mario.py
from cs50 import get_int

height = 0
while (height < 1 or height > 8):
    height = get_int("Height: ")

blankspaces = height
hashes = 0
for i in range(height):
    blankspaces -= 1
    hashes += 1
    print(" " * blankspaces, end="")
    print("#" * hashes, end="")
    print("")